src/plugins/discover/public/application/main/components/top_nav/get_top_nav_links.tsx

```diff
@@ -78,7 +78,6 @@ export const getTopNavLinks = ({
           defaultMessage: 'Try ES|QL',
         }),
     emphasize: true,
-    iconType: 'editorRedo',
     fill: false,
```

Comment:
The `editorRedo` icon is feeling extraneous and adds to the visual busy-ness. Further, it may be confusing alongside the `editorUndo` icon in the unsaved changes badge menu.

![CleanShot 2024-08-06 at 09 09 25@2x](https://github.com/user-attachments/assets/41d8c6b8-3d39-4021-9d8c-0e8fdeea3324)